src/Microsoft.SqlTools.ServiceLayer/Security/AppRoleGeneral.cs

```diff
@@ -38,40 +36,10 @@ internal class AppRoleGeneral
         //property by the initialization code
         private ServerConnection serverConnection;
 
+        #endregion
 
-        /// <summary>
-        /// execution mode by default for now is success
-        /// </summary>
-        private ExecutionMode m_executionMode = ExecutionMode.Success;
```

Comment:
I think @caohai is currently working on Roles, so it's possible he neither needs something here or has it already deleted in a feature branch.  Though from a quick pass these look unneeded to me.

Reply:
Thanks @kburtram ! Yeah I'm working on this part and I was planning to remove unused code with my next PR.